conf/defaults.ini

```diff
@@ -141,7 +141,7 @@ cookie_name = grafana_sess
 # If you use session in https only, default is false
 cookie_secure = false
 
-# Session life time, default is 86400
+# Session life time, default is 86400 (means 86400 seconds or 24 hours)
```

Comment:
this should be in sample.ini as well as this file is not something users usually view